async_fuse/src/main.rs

```diff
@@ -284,6 +293,17 @@ fn main() -> anyhow::Result<()> {
         None => CACHE_DEFAULT_CAPACITY,
     };
 
+    smol::spawn(Compat::new(async move {
```

Comment:
Please move this part to the metrics internal so that the metrics service only takes one line in the main function.

Reply:
DONE.